Vagrantfile

```diff
@@ -0,0 +1,41 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = '2'
+
+@script = <<SCRIPT
+apt-get update
+apt-get install -y apache2 git php5-cli php5 libapache2-mod-php5
+echo "
+<VirtualHost *:80>
+    ServerName zf2-tutorial.localhost
+    DocumentRoot /var/www/zf/public
+    SetEnv APPLICATION_ENV "development"
+    <Directory /var/www/zf/public>
+        DirectoryIndex index.php
+        AllowOverride All
+        Order allow,deny
+        Allow from all
+    </Directory>
+</VirtualHost>
+" > /etc/apache2/sites-available/000-default.conf
+a2enmod rewrite
+service apache2 restart
+cd /var/www/zf
+php composer.phar self-update
+COMPOSER_VENDOR_DIR=/opt/vendor php composer.phar install
+rm -rf /var/www/zf/vendor
```

Comment:
rm operations in repo folder is most than a simple code smell.

You should think how this script works the first day and the day 100th. iMay at dsay 100th custom vendor files are added and should not be removed (I need to do it in a project) Not everything is composer :)